packages/suite-desktop-core/e2e/tests/trading/swap-coin-to-token.test.ts

```diff
@@ -97,7 +97,7 @@ test.describe('Trading - Swap coin to token', { tag: ['@group=trading', '@webOnl
             await expect(tradingPage.confirmationCryptoAmount.last()).toHaveText(
                 formattedReceiveAmount,
             );
-            await expect(tradingPage.confirmationExchangeType).toHaveText('Fixed-rate offer');
+            await expect(tradingPage.confirmationExchangeType).toHaveText('Fixed');
```

Comment:
can we use messages here so that it does not fail when crowdin is changed? @Vere-Grey